internal/collector/persistentvolume.go

```diff
@@ -108,6 +108,21 @@ var (
 				}
 			}),
 		},
+		{
+			Name: "kube_persistentvolume_capacity",
+			Type: metric.MetricTypeGauge,
+			Help: "persistentvolume capacity in Gigabytes.",
```

Comment:
prometheus metrics best practice is to use base units, so this should be bytes (we're not loosing precision because we're using float64)